onnxruntime/core/framework/allocation_planner.cc

```diff
@@ -1761,6 +1761,44 @@ class PlannerImpl {
     num_logic_streams_ = stream_nodes_.size();
   }
 
+  // Construct WaitOnEPStep if has EP registered it.
+  // <param name="node">Current node to inspect</param>
+  // <param name="producer_node">producer node of the current node</param>
+  // <param name="notification_owner_index">index to find the corresponding stream owns the notification</param>
+  // <param name="stream_handle_registry">stream handle registry to get the wait handler</param>
+  // <param name="execution_plan">execution plan</param>
+  // <param name="stream_index">index to get current logic stream</param>
+  // <param name="devices_to_wait_on">Cached set of the devices to wait on</param>
+  void ConstructWaitOnEPStep(const Node* node, const Node& producer_node, size_t notification_owner_index,
```

Comment:
static function?

Reply:
As I replied in your earlier comment, we cannot make it static as it uses some instance variables like plan_